### PR TITLE
Fix Mermaid render problem

### PR DESCRIPTION
**Pull request by chrismetz09**
new head.html
```
<script src="/js/jquery-3.6.0.min.js"  crossorigin="anonymous"></script>
<script src="/js/popper-1.16.1.min.js" crossorigin="anonymous"></script>
<script src="/js/bootstrap-4.6.1.min.js" crossorigin="anonymous"></script>
<script src="{{ "js/script.js" | relURL }}"></script>
<script async src="/js/mermaid.min.js" crossorigin="anonymous"></script>
```

But sidebar did not appear so jquery remains there as well.

Wondering if moving scripts out of <head> to end of <body> could help. Tested well on localhosts for chrome and firefox.

/do-not-merge

### CODE REVIEW

**Comment by chrismetz09** [comment]
attempting to root cause and solution to [broken mermaid diagrams](https://github.com/kubernetes/website/issues/31960)

**Comment by chrismetz09** [comment]
https://deploy-preview-34794--kubernetes-io-main-staging.netlify.app/docs/contribute/style/diagram-guide/

**Comment by chrismetz09** [comment]
Updates:
1. running hugo 93.0, mermaid 18.3.4 and current jquery, popper and bootstrap packages.
2. Played around with more script positioning combinations.
3. Combo that reliably works in chrome and firefox:
`head.html` partial contains` jquery`, `mermaid` (if mermaid shortcode is present) and `script.js`. These scripts end up inside the `<head>` tags.

`scripts.html` partial contains `jquery`, `popper`, `bootstrap`, `script.js` and `Mermaid`. 

4. Devtools shows all scripts load once except for `mermaid.min.js`.

**Comment by reylejano** [comment]
Seems to be working for me
Example preview
https://deploy-preview-34794--kubernetes-io-main-staging.netlify.app/docs/concepts/services-networking/ingress/

File: content/en/docs/contribute/style/diagram-guide.md [hunk 1]
```diff
@@ -242,8 +207,12 @@ Figure 17. A to B
 more text
 ```
 {{< note >}}
+<<<<<<< HEAD
+You must include the Hugo Mermaid shortcode tags at the start and end of the Mermaid code block. You should add a diagram
+=======
 You must include the Hugo Mermaid shortcode
 tags at the start and end of the Mermaid code block. You should add a diagram
+>>>>>>> 580c643e50ac2aaac0b1b8f8781716a9429c003b
```
Comment (Sea-n) [inline content/en/docs/contribute/style/diagram-guide.md:215]: Merge conflict.

File: config.toml [hunk 2]
```diff
@@ -291,6 +291,7 @@ no = 'Sorry to hear that. Please <a href="https://github.com/USERNAME/REPOSITORY
 	icon = "fab fa-stack-overflow"
 	desc = "Practical questions and curated answers"
 
+
```
Comment (Sea-n) [inline config.toml:294]: Why add newline here?

**Review by Sea-n — COMMENTED** [review]
Seems there are git conflict to be resolved.

File: content/en/docs/contribute/style/diagram-guide.md (hunk 1 above)
Comment (chrismetz09) [reply content/en/docs/contribute/style/diagram-guide.md:215]: Hi @Sea-n, thanks for reviewing. This PR is for experimenting and testing inconsistent Mermaid diagram rendering. I will resolve merge conflict and Netlify deploy. If you can, check some of the pages with Mermaid diagrams (I will post in this PR). Once we see consistent results, I can close this PR and hack up a new one that will be merged and solve the problem.

Tx again.

Copying @sftim

**Comment by Babapool** [comment]
The diagrams are rendering for me. Thanks.

**Comment by Sea-n** [comment]
Hey @chrismetz09,

As PR author, you can retitle using GitHub Web UI anytime, no need to use the bot command.

**Review by sftim — COMMENTED** [review]
After these changes, Mermaid is loading reliably for me in Chromium (103.0.5060.134, Ubuntu amd64) and Firefox (103.0.1, Ubuntu amd64)

**Comment by sftim** [comment]
/label tide/merge-method-squash

(even better: squash locally, force-push, remove that label)

**Comment by tengqm** [comment]
@chrismetz09 Is this ready to be merged?

File: layouts/partials/head.html [hunk 3]
```diff
@@ -39,10 +39,6 @@
 {{ template "_internal/google_analytics_async.html" . }}
 {{ end }}
 {{ partialCached "head-css.html" . "asdf" }}
-<script
-  src="{{ "js/jquery-3.3.1.min.js" | relURL }}"
-  integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
```
Comment (sftim) [inline layouts/partials/head.html:44]: Are we losing this `integrity` value from the replacement code?

File: layouts/partials/scripts.html [hunk 4]
```diff
@@ -1,11 +1,12 @@
+<script src="/js/jquery-3.6.0.min.js"  crossorigin="anonymous"></script>
+<script src="/js/popper-1.16.1.min.js" crossorigin="anonymous"></script>
+<script src="/js/bootstrap-4.6.1.min.js" crossorigin="anonymous"></script>
+<script src="{{ "js/script.js" | relURL }}"></script>
+<script async src="/js/mermaid.min.js" crossorigin="anonymous"></script>
 
-<script src="/js/popper-1.14.3.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
-<script src="/js/bootstrap-4.3.1.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
```
Comment (sftim) [inline layouts/partials/scripts.html:3]: Are we losing this `integrity` value from the replacement code? If so, why is that?

**Comment by sftim** [comment]
/remove-label tide/merge-method-squash
/area web-development

/approve
/hold

This hold is to verify that as author, @chrismetz09 is happy to see this merge. @chrismetz09 you can remove this hold whenever you wish provided you are happy for the changes to land.

These changes LGTM (look good to me), even with the open question about `integrity`. Same-site `integrity` checks don't seem essential.

File: layouts/partials/scripts.html (hunk 4 above)
Comment (chrismetz09) [reply layouts/partials/scripts.html:3]: We are. Seem to recall something said it was okay to omit. I'll fix stat. Tx.

**Comment by sftim** [comment]
@chrismetz09 happy to unhold? Your call.

File: layouts/partials/head.html [hunk 5]
```diff
@@ -89,10 +85,12 @@
 <meta property="og:type" content="{{ $ogType }}">
 
 {{ if .HasShortcode "mermaid" }}
-<!-- Copied from https://unpkg.com/mermaid@8.11.2/dist/mermaid.min.js -->
+<!-- Copied from https://unpkg.com/mermaid@8.13.4/dist/mermaid.min.js -->
 <script async src="{{ "js/mermaid.min.js" | relURL }}"></script>
```
Comment (Sea-n) [inline layouts/partials/head.html:89]: I think it's better to move the file and include the version in the filename. It would be frustrating if we had to deal with bugs caused by caching problems.

```suggestion
<!-- Copied from https://unpkg.com/mermaid@8.13.4/dist/mermaid.min.js -->
<script async src="{{ "js/mermaid-8.13.4.min.js" | relURL }}"></script>
```

File: layouts/partials/scripts.html (hunk 4 above)
Comment (Sea-n) [reply layouts/partials/scripts.html:3]: (Remember to mark this conversion as solved, thanks.)

File: layouts/partials/scripts.html [hunk 6]
```diff
@@ -1,11 +1,12 @@
...
-
-{{ if .Site.Params.mermaid.enable }}
-<script src="/js/mermaid.min.js" crossorigin="anonymous"></script>
-
-{{ end }}
+<!--{{ if .Site.Params.mermaid.enable }}-->
+<!--<script async src="/js/mermaid.min.js" crossorigin="anonymous"></script>-->
+<!--{{ end }}-->
```
Comment (Sea-n) [inline layouts/partials/scripts.html:9]: Please just delete it, we have git history when we need those removed codes.

File: layouts/partials/head.html (hunk 5 above)
Comment (chrismetz09) [reply layouts/partials/head.html:89]: Very good idea but will save for future PR. I want to avoid changes that do not impact functionality at this point.

File: content/en/docs/contribute/style/diagram-guide.md [hunk 7]
```diff
@@ -37,17 +37,6 @@ X --> V[Tips]
     classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
     class A,C,D,N,X,m,T,V box
     class S spacewhite
-
-%% you can hyperlink Mermaid diagram nodes to a URL using click statements
-
-click A "https://mermaid-js.github.io/mermaid-live-editor/edit/#eyJjb2RlIjoiZmxvd2NoYXJ0IExSXG4gICAgc3ViZ3JhcGggbVtNZXJtYWlkLmpzXVxuICAgIGRpcmVjdGlvbiBUQlxuICAgICAgICBTWyBdLS4tXG4gICAgICAgIENbYnVpbGQ8YnI-ZGlhZ3JhbXM8YnI-d2l0aCBtYXJrZG93bl0gLS0-XG4gICAgICAgIERbb24tbGluZTxicj5saXZlIGVkaXRvcl1cbiAgICBlbmRcbiAgICBBW1doeSBhcmUgZGlhZ3JhbXM8YnI-dXNlZnVsP10gLS0-IG1cbiAgICBtIC0tPiBOWzMgeCBtZXRob2RzPGJyPmZvciBjcmVhdGluZzxicj5kaWFncmFtc11cbiAgICBOIC0tPiBUW0V4YW1wbGVzXVxuICAgIFQgLS0-IFhbU3R5bGluZzxicj5hbmQ8YnI-Y2FwdGlvbnNdXG4gICAgWCAtLT4gVltUaXBzXVxuICAgIFxuIFxuICAgIGNsYXNzRGVmIGJveCBmaWxsOiNmZmYsc3Ryb2tlOiMwMDAsc3Ryb2tlLXdpZHRoOjFweCxjb2xvcjojMDAwO1xuICAgIGNsYXNzRGVmIHNwYWNld2hpdGUgZmlsbDojZmZmZmZmLHN0cm9rZTojZmZmLHN0cm9rZS13aWR0aDowcHgsY29sb3I6IzAwMFxuICAgIGNsYXNzIEEsQyxELE4sWCxtLFQsViBib3hcbiAgICBjbGFzcyBTIHNwYWNld2hpdGUiLCJtZXJtYWlkIjoie1xuICBcInRoZW1lXCI6IFwiZGVmYXVsdFwiXG59IiwidXBkYXRlRWRpdG9yIjpmYWxzZSwiYXV0b1N5bmMiOnRydWUsInVwZGF0ZURpYWdyYW0iOnRydWV9" _blank
```
[the remark this answers is not in the document]
Comment (chrismetz09) [reply content/en/docs/contribute/style/diagram-guide.md:43]: Removed during troubleshooting because I wanted to streamline mermaid code inside the Hugo mermaid short code tags in case it might contribute to the problem.

[the remark this answers is not in the document]
Comment (Sea-n) [reply content/en/docs/contribute/style/diagram-guide.md:43]: I didn't get it. Could you explain more about what problem it caused?

File: layouts/partials/head.html [hunk 8]
```diff
@@ -89,10 +85,12 @@
...
 {{ end }}
```
Comment (Sea-n) [inline layouts/partials/head.html:90]: This PR upgraded `mermaid.js` from `v8.11.2` to `v8.13.4`. I think it's inappropriate to use the same filename.

Please revert this change or append the version number to the filename.

File: layouts/partials/scripts.html [hunk 9]
```diff
@@ -1,11 +1,8 @@
...
-
-{{ if .Site.Params.mermaid.enable }}
-<script src="/js/mermaid.min.js" crossorigin="anonymous"></script>
-
-{{ end }}
+<script src="/js/jquery-3.6.0.min.js" integrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" crossorigin="anonymous"></script>
+<script src="/js/popper-1.16.1.min.js" intregrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+<script src="/js/bootstrap-4.6.1.min.js" integrity="sha384-VHvPCCyXqtD5DqJeNxl2dtTyhF78xXNXdkwX1CZeRusQfRKp+tA7hAShOK/B/fQ2" crossorigin="anonymous"></script>
+<script src="{{ "js/script.js" | relURL }}"></script>
+<script async src="/js/mermaid.min.js" integrity="sha384-5hHNvPeMrNH14oM3IcQofDoBhiclNK3g2+hnEinKzQ07C4AliMeVpnvxuiwEGpaO" crossorigin="anonymous"></script>
```
Comment (Sea-n) [inline layouts/partials/scripts.html:5]: I'm not sure why it was `if .Site.Params.mermaid.enable` before.

Hey @sftim, do you think we should remove the `{{ if }}` here?

File: layouts/partials/scripts.html [hunk 10]
```diff
@@ -1,11 +1,8 @@
...
-<script src="/js/bootstrap-4.3.1.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
-
-{{ if .Site.Params.mermaid.enable }}
-<script src="/js/mermaid.min.js" crossorigin="anonymous"></script>
-
-{{ end }}
+<script src="/js/jquery-3.6.0.min.js" integrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" crossorigin="anonymous"></script>
+<script src="/js/popper-1.16.1.min.js" intregrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+<script src="/js/bootstrap-4.6.1.min.js" integrity="sha384-VHvPCCyXqtD5DqJeNxl2dtTyhF78xXNXdkwX1CZeRusQfRKp+tA7hAShOK/B/fQ2" crossorigin="anonymous"></script>
+<script src="{{ "js/script.js" | relURL }}"></script>
```
Comment (Sea-n) [inline layouts/partials/scripts.html:4]: Where did the `js/script.js` came from? It's still in `head.html`.

File: layouts/partials/head.html [hunk 11]
```diff
@@ -89,10 +85,12 @@
...
 
+<script src="/js/jquery-3.6.0.min.js" intregrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" crossorigin="anonymous"></script>
```
Comment (Sea-n) [inline layouts/partials/head.html:92]: Not sure why this PR moved `jquery` to the lower place.

**Review by Sea-n — COMMENTED** [review]
I'm not quite understand what's the major change of this PR.
There seems many unnecessary changes.

Can you explain why this PR will fix Mermaid render problem in your PR description? Thanks.

File: layouts/partials/head.html (hunk 8 above)
Comment (tengqm) [reply layouts/partials/head.html:90]: I'd vote for appending the version number to the filename.

File: layouts/partials/head.html (hunk 3 above)
Comment (chrismetz09) [reply layouts/partials/head.html:44]: added integrity SRIs

File: layouts/partials/head.html (hunk 8 above)
Comment (chrismetz09) [reply layouts/partials/head.html]: NP, made the change. I would like @sftim opinion on need to make changes that do not impact problem fix.

File: layouts/partials/scripts.html (hunk 9 above)
Comment (chrismetz09) [reply layouts/partials/scripts.html]: this code was inherited from docsy-example which was used during troubleshooting. Code has been removed.

File: layouts/partials/scripts.html (hunk 10 above)
Comment (chrismetz09) [reply layouts/partials/scripts.html:4]: My understanding is loading js libraries in the <head> tags can impact diagram rendering.

**Comment by chrismetz09** [comment]
Deploy preview failed. Checked log and it showed timeout. Not sure how to re-run Netlify deploy.

**Comment by jihoon-seo** [comment]
Netlify build failed with no particular reason.
@chrismetz09 please (force-)push again, then a new Netlify build will be triggered.

**Comment by sftim** [comment]
I've triggered a new build.

**Comment by sftim** [comment]
/hold cancel

I didn't see any reason for this PR to be held.

File: layouts/partials/head.html (hunk 8 above)
Comment (sftim) [reply layouts/partials/head.html]: When making unrelated or semi-related changes, I like to see those in their own commit (makes code history searches easier, or even reverts if we need to do that).

File: layouts/partials/head.html (hunk 11 above)
Comment (sftim) [reply layouts/partials/head.html:92]: @chrismetz09 was there a reason?

**Comment by jihoon-seo** [comment]
@chrismetz09 @sftim Netlify build failed again. 🤔

**Comment by jihoon-seo** [comment]
Citing [this](https://github.com/kubernetes/website/pull/34794#issuecomment-1202179670):

/label tide/merge-method-squash

(even better: squash locally, and force-push)

**Comment by chrismetz09** [comment]
Will complete 8.24

**Comment by sftim** [comment]
Try rebasing against the _main_ branch once v1.25 is out @chrismetz09 - the build should work OK.

**Comment by chrismetz09** [comment]
@sftim, need clarification on git commands to rebase against _main_ branch. Would these work? 
```
git checkout main
git pull origin main
git checkout bugMermaid
git rebase -i origin main
```

**Comment by sftim** [comment]
I'm assuming that your remote named _origin_ is https://github.com/kubernetes/website/ @chrismetz09 

You might have meant:
```shell
git checkout main
git pull origin main
git checkout bugMermaid
git rebase -i origin/main
```

Before you start this, check if you have any pending changes (`git status`) and then tidy those away if you do (`git stash --include-untracked` for example).

**Comment by chrismetz09** [comment]
Rebased to main. Final command is ?:
```
git push --force-with-lease origin bugMermaid
```

**Comment by sftim** [comment]
What's the name of the remote representing your fork of the repo, @chrismetz09 ?

`git push --force-with-lease metZ bugMermaid` looks right to me

**Review by sftim — COMMENTED** [review]
I'm not sure I understand why there are changes to `content/en/docs/contribute/style/diagram-guide.md`, and to various diagrams.

@chrismetz09 can you clarify?

**Comment by chrismetz09** [comment]
Reasons for Diagram Guide mods:
- DG contains lots of Mermaid in-line diagrams I used for testing.
- removed Mermaid click events for the DG diagrams to streamline in-line mermaid code. I was thinking they could be problem cause. They were not. 
- Colleague somewhere (Zach?) wondered about persisting examples in the live-editor that click events point to.

**Comment by sftim** [comment]
> removed Mermaid click events for the DG diagrams to streamline in-line mermaid code. I was thinking they could be problem cause. They were not.

I'm fine to remove them, but I would like that to be in a different commit to the Mermaid fix.

File: layouts/partials/head.html (hunk 11 above)
Comment (chrismetz09) [reply layouts/partials/head.html:81]: early troubleshooting task exploring moving script files to end of file.

**Comment by chrismetz09** [comment]
> > removed Mermaid click events for the DG diagrams to streamline in-line mermaid code. I was thinking they could be problem cause. They were not.
> 
> I'm fine to remove them, but I would like that to be in a different commit to the Mermaid fix.

NP. Not sure on procedures to achieve this. Found something like this that could do the trick? Feature branch is bugMermaid and I want to remove diagram-guide.md file from the PR.

```
website git:(bugMermaid) git checkout origin/main -- ./diagram-guide.md
website git:(bugMermaid) git add ..something..
website git:(bugMermaid) git commit -m "remove diagram.md changes from PR"
website git:(bugMermaid)  git push --force-with-lease metZ bugMermaid 
```

File: layouts/partials/scripts.html [hunk 12]
```diff
@@ -1,11 +1,8 @@
-
-<script src="/js/popper-1.14.3.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
-<script src="/js/bootstrap-4.3.1.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
-
-{{ if .Site.Params.mermaid.enable }}
-<script src="/js/mermaid.min.js" crossorigin="anonymous"></script>
-
-{{ end }}
+<script src="/js/jquery-3.6.0.min.js" integrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" crossorigin="anonymous"></script>
+<script src="/js/popper-1.16.1.min.js" intregrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
```
Comment (sftim) [inline layouts/partials/scripts.html:2]: What's the source for this file? (I can't get the subresource integrity check to come out right).

File: layouts/partials/scripts.html [hunk 13]
```diff
@@ -1,11 +1,8 @@
...
+<script src="/js/bootstrap-4.6.1.min.js" integrity="sha384-VHvPCCyXqtD5DqJeNxl2dtTyhF78xXNXdkwX1CZeRusQfRKp+tA7hAShOK/B/fQ2" crossorigin="anonymous"></script>
```
Comment (sftim) [inline layouts/partials/scripts.html:3]: What's the source for this file? (I can't get the subresource integrity check to come out right).

File: layouts/partials/scripts.html (hunk 10 above)
Comment (sftim) [reply layouts/partials/scripts.html:4]: If we omit `js/script.js` from one of the **two** places where it's included, does each page still load OK?

File: layouts/partials/scripts.html (hunk 12 above)
Comment (chrismetz09) [reply layouts/partials/scripts.html:2]: Are these appropriate steps to doublecheck SRI?

1. Get the `.min.js` file from the CD.
2. For this file, generate SRI using `create-sri` and the `openssl ...` methods.
3. Go to ./static/js folder and repeat steps for local `.min.js` files.
4. Verify generated SRI values match integrity field in the code.

I tried it with jquery and they seem to line up.

---

Checking Jquery SRI with the following steps. Generates SRI.
```
Get the file:
curl -o jquery-3.6.0.min.js https://code.jquery.com/jquery-3.6.0.min.js

Check SRI:
➜  ~ create-sri jquery-3.6.0.min.js
sha384 encryption result:
sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK

Check SRI:
➜  ~ cat jquery-3.6.0.min.js | openssl dgst -sha384 -binary | openssl base64 -A
vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK%
```
---

Change to ./static/js folder to check SRI for jquery-3.6.0.min.js. 
```
➜  js git:(bugMermaid) create-sri jquery-3.6.0.min.js
sha384 encryption result:
sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK

➜  js git:(bugMermaid) cat jquery-3.6.0.min.js | openssl dgst -sha384 -binary | openssl base64 -A
vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK%
```

In ./partials/head.html
```
<script src="/js/jquery-3.6.0.min.js" intregrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" crossorigin="anonymous"></script>
```
In ./partials/scripts.html
```
<script src="/js/jquery-3.6.0.min.js" integrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" crossorigin="anonymous"></script>
```

File: layouts/partials/scripts.html (hunk 13 above)
Comment (chrismetz09) [reply layouts/partials/scripts.html:3]: Popper file.
```
curl -o popper-1.16.1.min.js https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js

~ create-sri popper-1.16.1.min.js
sha384 encryption result:
sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN

 ~ cat popper-1.16.1.min.js | openssl dgst -sha384 -binary | openssl base64 -A
9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN%
```

Same file in ./static/js folder.
```
js git:(bugMermaid) create-sri popper-1.16.1.min.js
sha384 encryption result:
sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN

js git:(bugMermaid) cat popper-1.16.1.min.js | openssl dgst -sha384 -binary | openssl base64 -A
9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN%
```
Code in `scripts.html`
```
<script src="/js/popper-1.16.1.min.js" intregrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
```

Comment (chrismetz09) [reply layouts/partials/scripts.html:3]: Bootstrap file.
```
 ~ curl -o bootstrap-4.6.1.min.js https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.min.js

 ~ cat bootstrap-4.6.1.min.js | openssl dgst -sha384 -binary | openssl base64 -A
VHvPCCyXqtD5DqJeNxl2dtTyhF78xXNXdkwX1CZeRusQfRKp+tA7hAShOK/B/fQ2%
 ```
 
 same file in ./static.js folder
 ```
 cat bootstrap-4.6.1.min.js | openssl dgst -sha384 -binary | openssl base64 -A
VHvPCCyXqtD5DqJeNxl2dtTyhF78xXNXdkwX1CZeRusQfRKp+tA7hAShOK/B/fQ2%
 ```
 Code in scripts.html
 ```
 <script src="/js/bootstrap-4.6.1.min.js" integrity="sha384-VHvPCCyXqtD5DqJeNxl2dtTyhF78xXNXdkwX1CZeRusQfRKp+tA7hAShOK/B/fQ2" crossorigin="anonymous"></script>
 ```

**Comment by chrismetz09** [comment]
Removed `/js/script.js` from `scripts.html` and page loads seem okay. Should I commit it?

**Comment by sftim** [comment]
Thanks for letting me know the URIs. Provenance checks out.

/lgtm

@chrismetz09 - that removal can be a follow-up PR!
